src/components/App/App.component.test.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App.component';
+
+describe('Render App', () => {
+  test('it should render Welcome to the Challenge!', () => {
+    render(<App />);
+    expect(screen.getByText('Welcome to the Challenge!')).toBeInTheDocument();
+  });
+});
```

Comment:
I normally would not add a test for the `App` file since that would not be the purpose of an unit test but an integration test. As long as you test the separate units of you're app, you can leave the whole integration to another testing strategy.